packages/react-dom/src/useInView.tsx

```diff
@@ -44,6 +48,11 @@ export function useInView({
       (inView, entry) => {
         setState({ inView, entry })
         onChangeRef.current?.(inView, entry)
+        if (inView) {
+          onInView?.(entry)
+        } else {
+          onInViewEnd?.(entry)
+        }
```

Comment:
I’m wondering if onInViewEnd gets triggered on mount when the element is initially outside the viewport?

Reply:
Oh... You save me! Thanks

Reply:
I fixed it